package.json

```diff
@@ -174,6 +173,7 @@
 		"vscode-uri": "^3.0.6"
 	},
 	"devDependencies": {
+		"@edfi/ed-fi-model-5.0": "3.0.0",
```

Comment:
@jleiva-gap looking at the diff here, it looks like you just moved the 5.0 dependency from `dependencies` to `devDependencies`, whereas the original line 146 was already correct.

Reply:
Oh, I see, It was moved when I run the npm install, so I think we can close this PR, the repo is already updated, I didn't need to add anything else to this repo.